windows启动报错: panic: mcp time config templates example is invalid

版本: Version 0.4.9
系统: windows11
启动时报错,无法运行

C:\Users\lemcon>C:\Users\lemcon\Downloads\aiproxy-windows-amd64.exe
panic: mcp time config templates example is invalid: config local_timezone validate error: example is invalid: unknown time zone America/New_York

goroutine 1 [running]:
github.com/labring/aiproxy/mcp-servers.Register({{{0x7ff77452e8d1, 0x4}, {0x0, 0x0, 0x0}, {0x0, 0x0, 0x0}, {0x0, 0x0, ...}, ...}, ...})
        github.com/labring/aiproxy/mcp-servers/register.go:95 +0x3bc
github.com/labring/aiproxy/mcp-servers/hosted/time.init.0()
        github.com/labring/aiproxy/mcp-servers/hosted/time/init.go:17 +0x331

<img width="1728" height="923" alt="Image" src="https://github.com/user-attachments/assets/2ee777db-ce8b-41e0-ba27-b482b5f984a4" />

--- a/mcp-servers/hosted/time/time.go
+++ b/mcp-servers/hosted/time/time.go
@@ -6,6 +6,7 @@ import (
 	"fmt"
 	"strings"
 	"time"
+	_ "time/tzdata"
 
 	"github.com/bytedance/sonic"
 	mcpservers "github.com/labring/aiproxy/mcp-servers"

--- a/mcp-servers/hosted/time/time_test.go
+++ b/mcp-servers/hosted/time/time_test.go
@@ -1,0 +1,23 @@
+package time
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestLocalTimezoneConfigTemplateExampleIsValid(t *testing.T) {
+	t.Parallel()
+
+	template := configTemplates["local_timezone"]
+	require.NotNil(t, template.Validator)
+	require.NoError(t, template.Validator(template.Example))
+}
+
+func TestGetTimezoneSupportsIANAName(t *testing.T) {
+	t.Parallel()
+
+	loc, err := getTimezone("America/New_York")
+	require.NoError(t, err)
+	require.Equal(t, "America/New_York", loc.String())
+}

